README.md

```diff
@@ -1,2 +1,254 @@
 # smartclide-context
 SmartCLIDE Context Handling Component
+
+## Preconditions to build and run Context Handling
+
+To build and run Context Handling, the following software is required:#
+
+- Java (at least version 11)
+- Apache Maven (min V3.6)
+- Docker [Desktop] (for running Context Handling)
+
+## How to build Context Handling
+
+Context Handling can be built using maven with the following command:
+
+  `mvn package`
+
+In order to build a docker container image that can be deployed to a docker host, the following command can be used:
+
+  `mvn clean ...`
+
+Note: Currently the configuration files are packed into the container images, which means that on each configuration change a new container has to be created and deployed.
+
+## How to run Context Handling
+
+There are two possibilites to execute Context Handling. 
+
+1. Building and runnning the locally built Context Handling can be done using the following command:
+   ```
+   buildAndRunCHLocally.sh
+   ```
+
+2. Executing Context Handling using the Docker container image from the ATB container registry:
+   ```
+   docker run ....
+   ```
+   
+## How to configure Context Handling
+
+The Context Handling configuration files are store in the following folder:
+`./resources/`
+
+### Monitoring Config
+
+In the following the xml schema for the monitoring configuration is listed. Below the xsd the configuration elements are described in detail.
+
+**monitoring-config.xsd**
```

Comment:
I would suggest to just link to the XSD files from here. That way the README content cannot get out of sync with the actual XSD content.

Reply:
[x] done